legal-api/src/legal_api/reports/business_document.py

```diff
@@ -499,6 +501,54 @@ def _set_liquidation_details(self, business: dict):
                 liquidation_info['recordsOffice'] = records_office_info
         business['liquidation'] = liquidation_info
 
+    def _set_continuation_in_details(self, business: dict):
+        """Set continuation in filing data."""
+        continuation_in_info = {}
+        continuation_in_filing = Filing.get_filings_by_types(self._business.id, ['continuationIn'])
+        if continuation_in_filing:
+            continuation_in_filing = continuation_in_filing[0]
+            continuation_in_data = continuation_in_filing.filing_json['filing']['continuationIn']
+
+            continuation_in_info['filingDateTime'] = continuation_in_filing.filing_date.isoformat()
+            continuation_in_info['mode'] = continuation_in_data.get('mode', 'N/A')
+
+            # Processing offices
+            continuation_in_info['offices'] = continuation_in_data['offices']
+            for office_data in continuation_in_data['offices'].values():
+                self._format_address(office_data['mailingAddress'])
+                self._format_address(office_data['deliveryAddress'])
+
+            # Processing parties
+            continuation_in_info['parties'] = continuation_in_data['parties']
+            for party in continuation_in_data['parties']:
+                self._format_address(party['mailingAddress'])
+                self._format_address(party['deliveryAddress'])
+
+            continuation_in_info['courtOrder'] = continuation_in_data.get('courtOrder', {})
+            continuation_in_info['nameRequest'] = continuation_in_data.get('nameRequest', {})
+            continuation_in_info['contactPoint'] = continuation_in_data.get('contactPoint', {})
+            continuation_in_info['authorization'] = continuation_in_data.get('authorization', {})
+            continuation_in_info['shareStructure'] = continuation_in_data.get('shareStructure', {})
+            continuation_in_info['nameTranslations'] = continuation_in_data.get('nameTranslations', [])
+            continuation_in_info['foreignJurisdiction'] = continuation_in_data.get('foreignJurisdiction', {})
```

Comment:
can get this from https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/models/jurisdiction.py#L55